.pre-commit-config.yaml

```diff
@@ -39,13 +39,13 @@ repos:
         files: ^(vue-model-api)|(model-api-gen-gradle-test/vue-integration)/
   - repo: https://github.com/pre-commit/mirrors-eslint
     # IMPORTANT: keep versions of hook and additional dependencies in sync with the versions configured in `vue-model-api/package.json`
-    rev: v8.50.0
+    rev: v8.56.0
     hooks:
       - id: eslint
         additional_dependencies:
           - "eslint@8.56.0"
           - "eslint-config-prettier@9.1.0"
-          - "@typescript-eslint/eslint-plugin@7.0.1"
-          - "@typescript-eslint/parser@7.0.1"
+          - "@typescript-eslint/eslint-plugin@7.4.0"
```

Comment:
I am still unsure, if I want to run Prettier and ESLint with pre-commit.
(With Renovate we aim to have the diverging version under control automatically.)
Or if it is better just to run it through NPM and Gradle.
I currently tend to leave it in pre-commit because it fails faster and gives earlier feedback.